README.md

```diff
@@ -96,6 +96,52 @@ piccolo asgi new
 
 [Starlette](https://www.starlette.io/), [FastAPI](https://fastapi.tiangolo.com/), [BlackSheep](https://www.neoteroi.dev/blacksheep/), [Litestar](https://litestar.dev/), [Esmerald](https://esmerald.dev/), [Lilya](https://lilya.dev), [Quart](https://quart.palletsprojects.com/en/latest/), [Falcon](https://falconframework.org/) and [Sanic](https://sanic.dev/en/) are currently supported.
 
+## Piccolo ecosystem
+
+### Piccolo Admin
+
+Piccolo Admin is a powerful admin interface / content management system for Python, built on top of Piccolo.
+
+![Screenshot](https://raw.githubusercontent.com/piccolo-orm/piccolo_admin/master/docs/images/screenshot.png "Screenshot")
+
+It was created at a design agency to serve the needs of customers who demand a high quality, beautiful admin interface for their websites. It's a modern alternative to tools like Wordpress and Django Admin.
+
+It's built using the latest technologies, with Vue.js on the front end, and a powerful REST backend.
+
+Some of it's standout features:
+
+* Powerful data filtering
+* Builtin security
+* Multi-factor Authentication
+* Media support, both locally and in S3 compatible services
+* Dark mode support
+* CSV exports
+* Easily create custom forms
+* Works on mobile and desktop
+* Use standalone, or integrate with several supported ASGI frameworks
+* Multilingual out of box
+* Bulk actions, like updating and deleting data
+* Flexible UI - only show the columns you want your users to see
+
+You can read the docs [here](https://piccolo-admin.readthedocs.io/en/latest/).
+
+### Piccolo API
+
+Utilities for easily exposing [Piccolo](https://piccolo-orm.readthedocs.io/en/latest/) tables as REST endpoints in ASGI apps, such as [Starlette](https://www.starlette.io) and [FastAPI](https://fastapi.tiangolo.com/).
+
+Includes a bunch of useful ASGI middleware:
+
+- Session Auth
+- Token Auth
+- Rate Limiting
+- CSRF
+- Content Security Policy (CSP)
+- And more
+
+It also contains excellent Pydantic support, allowing you to easily create Pydantic models based on your Piccolo tables.
```

Comment:
This isn't really the case any more - we move this into the main Piccolo repo.

Reply:
Yes, you're right and this was moved to the main Piccolo repo a long time ago. I'll leave it up to you to update the Piccolo API readme as well. I will remove this from the PR.